flask_admin/contrib/mongoengine/view.py

```diff
@@ -545,6 +565,37 @@ def update_model(self, form, model):
 
         return True
 
+    def update_list_model(self, form):
+        """
+            Update model from the list view.
+            
+            Only supports updating a single field at a time.
+
+            :param form:
+                Form instance
+        """
+        try:
+            model = self.model()
+            for field in form:
+                # FieldList's last_index will only be set if a field is submitted
+                # last_index will be the primary key of the updated record
+                if getattr(field, 'last_index', None):
+                    record = self.get_one(field.last_index)
```

Comment:
I ended up completely getting rid of this function. I found a way to use `update_model` instead of creating a new function.